core/stat/base/sliding_window_metric.go

```diff
@@ -159,7 +159,7 @@ func (m *SlidingWindowMetric) SecondMetricsOnCondition(predicate base.TimePredic
 	ws := m.real.ValuesConditional(util.CurrentTimeMillis(), predicate)
 
 	// Aggregate second-level MetricItem (only for stable metrics)
-	wm := make(map[uint64][]*BucketWrap)
+	wm := make(map[uint64][]*BucketWrap, 4)
```

Comment:
maybe len(ws) is better?

Reply:
The key changes to `start second`,  many items  in `ws` may have same key in `wm`, it it hard to predict the number of keys in `wm`.